app/templates/_labtemplates/_sass/pattern-scaffolding.scss

```diff
@@ -15,7 +15,7 @@
 }
 
 .demo-animate {
-  background: #dddddd;
+  background: #ddd;
```

Comment:
dit is niet conform `supple-config-stylelint`, gebruik je die wel?

Reply:
Ik wil hex-notatie matchen met Zeplin/Sketch/Abstract's notatie.

Reply:
Sketch laat het gewoon in 6 karakters zien toch?

Reply:
Oh ik heb het de verkeerde kant op veranderd. Zo?
```
"color-hex-length": "long",
"color-hex-case": "upper",
```